templates/ado-resources/TODO.md

```diff
@@ -0,0 +1,41 @@
+# TODO : Azure DevOps Pipeline
+
+Once you have created this template, to activate the pipeline in Azure DevOps, you will need to:
+
+1. Validate that a variable group name `CloudflarePages` exists in the Azure DevOps project.
+   1. Go to `Pipelines > Library`. From there you can create or edit the variable group `CloudflarePages`.
+   2. Add the following variables (more details on how to get the values [here](https://developers.cloudflare.com/workers/wrangler/ci-cd/)):
+      1. `CLOUDFLARE_ACCOUNT_ID` : the Cloudflare account ID. Must be set as `secret`.
+      2. `CLOUDFLARE_API_TOKEN` : the Cloudflare API token. Must be set as `secret`.
+2. For the pipeline to work, you also need to install the extension `PR Messenger` in your Azure DevOps organization.
+   1. Install the extension `PR Messenger` from the [marketplace](https://marketplace.visualstudio.com/items?itemName=abeaudoin.pr-messenger).
```

Comment:
Could the extension in the market place be named under "Workleap" rather than "abeaudoin" ?

Reply:
good catch

Reply:
Yes! I will look at it and add you two to the publisher at the same time